synapse/lib/scope.py

```diff
@@ -64,8 +64,8 @@ def get(self, name, defval=None):
         Retrieve a value from the closest scope frame.
         '''
         for frame in reversed(self.frames):
-            valu = frame.get(name)
-            if valu != None:
+            valu = frame.get(name, defval)
+            if valu != defval:
```

Comment:
True... only if there exists a parent frame containing a prop with _name_ it will find that and miss the _real_ value in the child frame. Thanks for pointing out 'novalu', def a much better approach. I added a unit test to cover the failure case you brought up.